### ISSUE TITLE
Marigo2017 isochrones have incorrect values

### ISSUE THREAD

**Issue by apace7** [DarkEnergySurvey/ugali#29]
The Marigo2017 isochrone tarball has two points which are incorrect.  One looks to be the last data point in the file which has a discontinuity to the other points.

[test.pdf](https://github.com/DarkEnergySurvey/ugali/files/1419600/test.pdf)

**Comment by kadrlica** [comment]
Can you confirm whether you are plotting `Marigo2017.mag_1` and `Marigo2017.mag_2` or the output of `Marigo2017.sample`?

**Comment by kadrlica** [comment]
Do we know if these points are actually unintended or are they an artifact of trying to include post-AGB objects?

**Comment by apace7** [comment]
One of the data point is likely artifact (at M~30).  The last point in every file has this problem.  Looking at a couple of the files the last data point has logg = -9.999.  

I think the other point that is disjoint is for the AGB stars.

**Comment by kadrlica** [comment]
I've looked at the affected file. If I'm counting columns correctly, I think it's `logL = -9.999` (not `logg`). This is at least consistent with the mag ~ 30 values). I've also noticed that `stage = 9`, which I do not see defined as a valid stage [here](http://stev.oapd.inaf.it/cmd_3.0/help.html).

I'm going to reach out to the isochrone maintainers and update `parsec.py` to reject points with `stage == 9`.
